schematic/models/validate_attribute.py

```diff
@@ -868,186 +883,590 @@ def url_validation(
+
+            # Do the validation on both columns
+            if "matchNone" in val_rule:
+                # Look for repeats between the source manifest and target_column, if there are
+                # repeats log the repeat value and manifest
+                repeat_values = manifest_col[manifest_col.isin(target_column)]
+
+                if repeat_values.any():
+                    repeat_manifest_log[target_manifest_id] = repeat_values
+            else:
+                # Determine elements in manifest column that are missing from the target column
+                missing_values = manifest_col[~manifest_col.isin(target_column)]
+
+                if missing_values.empty:
+                    # If there are no missing values in the target column, log this manifest as
+                    # one where all items are present
+                    present_manifest_log.append(target_manifest_id)
+                else:
+                    # If there are missing values in the target manifest, log the manifest
+                    # and the missing values.
+                    missing_manifest_log[target_manifest_id] = missing_values
+
+        return missing_manifest_log, present_manifest_log, repeat_manifest_log
+
+    def _gather_target_columns_value(
+        column_names: Dict[str, str],
+        target_attribute: str,
+        concatenated_target_column: pd.core.series.Series,
+        target_manifest: pd.core.series.Series,
+    ) -> pd.core.series.Series:
+        """
+        Args:
+            column_names, Dict: {stripped_col_name:original_column_name}
+            target_attribute, str: current target attribute
+            concatenated_target_column, pd.core.series.Series: target column in the process of being built, possibly
+                passed through this function multiple times based on the number of manifests
+            target_manifest, pd.core.series.Series: current target manifest
+        Returns:
+            concatenated_target_column, pd.core.series.Series: All target columns concatenated into a single column
+        """
+        # Gather information as needed to perform set/value cross manifest validation
+
+        if target_attribute in column_names:
+            target_manifest.rename(
+                columns={column_names[target_attribute]: target_attribute},
+                inplace=True,
+            )
+
+            if concatenated_target_column.any():
+                concatenated_target_column = pd.concat(
+                    objs=[
+                        concatenated_target_column,
+                        target_manifest[target_attribute],
+                    ],
+                    join="outer",
+                    ignore_index=True,
+                )
+            else:
+                concatenated_target_column = target_manifest[target_attribute]
+            concatenated_target_column = concatenated_target_column.astype("object")
+
+        return concatenated_target_column
```

Comment:
I'm having a really hard time figuring out what this method is doing. I would suggest some mroe comments.